### PR TITLE
Update user guide

### PR DESCRIPTION
**Pull request by hiltonlima**
* Added SDK Python test in TH User Interface session
* Fixed command to execute open SDK container
* Minor fix into Release History table
* Added PIXIT parameter information
* 
https://github.com/project-chip/certification-tool/blob/bdabd07e1ac4c17e03dac640665e2b99e3e4184d/docs/Matter_TH_User_Guide/Matter_TH_User_Guide.adoc

### CODE REVIEW

File: docs/Matter_TH_User_Guide/Matter_TH_User_Guide.adoc [hunk 1]
```diff
@@ -1261,17 +1257,67 @@ Execute the below command in the docker for the BLE+Thread pairing:
...
+----
+
+Considering this project configuration and selecting tests TC_ACE_1_1 and TC_ACE_1_2. 
+
+image:images/img_61.png[]
+
+This configuration will result in the following execution steps:
+. Perform DUT commissioning
+. Run the TC_ACE_1_2 test (without commissioning)
+. Run the TC_ACE_1_3 test (without commissioning)
```
Comment (ccruzagralopes) [inline docs/Matter_TH_User_Guide/Matter_TH_User_Guide.adoc:1306]: This is not being shown as a list

Screenshot 2023-12-14 at 15 54 19

**Review by antonio-amjr — APPROVED** [review]

File: docs/Matter_TH_User_Guide/Matter_TH_User_Guide.adoc [hunk 2]
```diff
@@ -1261,17 +1257,67 @@ Execute the below command in the docker for the BLE+Thread pairing:
...
+  },
+  "test_parameters": {
+    "paa-trust-store-path": "/credentials/development/paa_roots"
+    "storage-path": "admin_storage.json"
+  }
+  ...
+}
+----
+
+Considering this project configuration and selecting tests TC_ACE_1_1 and TC_ACE_1_2.
```
Comment (rquidute) [inline docs/Matter_TH_User_Guide/Matter_TH_User_Guide.adoc:1299]:
```suggestion
Considering this project configuration and selecting tests TC_ACE_1_2 and TC_ACE_1_3.
```

File: docs/Matter_TH_User_Guide/Matter_TH_User_Guide.adoc [hunk 3]
```diff
@@ -1261,17 +1257,67 @@ Execute the below command in the docker for the BLE+Thread pairing:
...
+}
+----
+
+Considering this project configuration and selecting tests TC_ACE_1_1 and TC_ACE_1_2. 
+
+image:images/img_61.png[]
+
+This configuration will result in the following execution steps:
+. Perform DUT commissioning
+. Run the TC_ACE_1_2 test (without commissioning)
```
Comment (rquidute) [inline docs/Matter_TH_User_Guide/Matter_TH_User_Guide.adoc:1305]:
```suggestion
. Run the TC_ACE_1_2 test (with commissioning)
```

Comment (hiltonlima) [reply docs/Matter_TH_User_Guide/Matter_TH_User_Guide.adoc:1305]: In this case is correct. We ran the tc_ace_1.2 without commissioning...

File: docs/Matter_TH_User_Guide/Matter_TH_User_Guide.adoc (hunk 1 above)
Comment (hiltonlima) [reply docs/Matter_TH_User_Guide/Matter_TH_User_Guide.adoc]: Done :)

**Review by rquidute — APPROVED** [review]

**Review by ccruzagralopes — APPROVED** [review]

File: docs/Matter_TH_User_Guide/Matter_TH_User_Guide.adoc [hunk 4]
```diff
@@ -1320,6 +1320,39 @@ These steps performed by TH are equivalent to the following steps when performed
  
  python3 TC_ACE_1_3.py --discriminator 3840 --passcode 20202021  --paa-trust-store-path /credentials/development/paa_roots --storage-path admin_storage.json
 
+===== PIXIT Support
+
+PIXIT type parameters must be filled in in the section (*test_parameters*).
```
Comment (ccruzagralopes) [inline docs/Matter_TH_User_Guide/Matter_TH_User_Guide.adoc:1325]:
```suggestion
PIXIT type parameters must be filled in the *test_parameters* section.
```

File: docs/Matter_TH_User_Guide/Matter_TH_User_Guide.adoc [hunk 5]
```diff
@@ -1320,6 +1320,39 @@ These steps performed by TH are equivalent to the following steps when performed
...
+    "paa-trust-store-path": "/credentials/development/paa_roots",
+    "storage-path": "admin_storage.json",
+    "int-arg": "PIXIT.ACE.APPENDPOINT:1 PIXIT.ACE.APPDEVTYPEID:256",
+    "string-arg": "PIXIT.ACE.APPCLUSTER:OnOff PIXIT.ACE.APPATTRIBUTE:OnOff"
+  }
+  ...
+}
+----
+
+The following example will be used to define the following parameters when running the test:
```
Comment (ccruzagralopes) [inline docs/Matter_TH_User_Guide/Matter_TH_User_Guide.adoc:1350]:
```suggestion
The above example will be used to define the following arguments when running the test:
```

**Pull request merged**